org-gtd-customize.el

```diff
@@ -76,14 +76,15 @@ into a datetree."
   :package-version '(org-gtd . "2.0.0"))
 
 (defcustom org-gtd-capture-templates
-  '(("i" "Inbox" "* %?\n%U\n\n  %i")
-    ("l" "Inbox with link" "* %?\n%U\n\n  %i\n  %a"))
+  '(("i" "Inbox"
```

Comment:
No precise reason, I just left it out of the commit as if it was personal preference.

Reply:
Gotcha. This means that right now the pull request is not just a change in implementation but actually a change in behavior. If we're going to do this, we need to ensure continuity for the baseline users, so we need to have capture templates with a finalize hook. I realize it's still going to change their experience, because they won't see the timestamp when they hit the capture, but an upgrade message can take care of that.